doc_src/prompt.rst

```diff
@@ -148,10 +148,50 @@ And it looks like:
 .. parsed-literal::
     :class: highlight
 
+    :green:`~/M/L/Oneknowing`>false
     :green:`~/M/L/Oneknowing`\ :red:`[1]`>_
```

Comment:
This is a nice example. I think we should even consider making this the default.

Our default prompt is a bit noisy given long pipelines that fail (due to the last command exiting with nonzero)

	user@localhost ~/g/fish-shell > echo foo | cat | cat | grep not-found
	user@localhost ~/g/fish-shell [0|0|0|1]> 

The exit status is probaly not super interesting to have stick around?

(Going further, I think collapsing the final rendering to a mere `$ ` would be a good idea for lots of use cases but that's another discussion)